fiddler/tests/test_fiddler.py

```diff
@@ -7,23 +7,160 @@
 from datadog_checks.dev.utils import get_metadata_metrics
 from datadog_checks.fiddler import FiddlerCheck
 
-from .conftest import FIDDLER_API_KEY
+from .conftest import (
```

Comment:
Imports from `conftest` aren't the conventional way to use that file. Could you turn your mocks into `pytest.fixture`s and use them that way please?

Reply:
Great point! Have fixed it.
Thanks for the review 🙏